site/app/controllers/HomePageController.php

```diff
@@ -40,6 +40,21 @@ public function showLibrary() {
 		);
     }
 
+    /**
+     * Display the LibraryManagerView to the instructor/admin.
+     *
+     * @Route("/homework/library/manage", methods={"GET"})
```

Comment:
Currently, this link is non-dynamic and sends the user back to the home page even if they have a course and semester selected. This should be changed to maintain the user's current course and semester information. 

See #79 on adding dynamic routes.

Reply:
We could also change the button to say something like "Go to library manager" to indicate that the user is leaving a course-specific section of the site.